core/base/src/TColor.cxx

```diff
@@ -64,6 +64,8 @@ The color creation and management class.
   - [Gray scale view of of canvas with colors](\ref C04)
   - [Color palettes](\ref C05)
   - [High quality predefined palettes](\ref C06)
+    - [Colour Vision Deficiency (CVD) friendly palettes](\ref C06a)
+    - [Non Colour Vision Deficiency (CVD) friendly palettes](\ref C06b)
```

Comment:
Is the change of indentation intentional?

Reply:
Yes they are subsections of the one above